iothub/service/src/Authentication/IotHubConnectionProperties.cs

```diff
@@ -33,8 +29,8 @@ protected IotHubConnectionProperties(string hostName)
 
             HostName = hostName;
             IotHubName = GetIotHubName(hostName);
-            AmqpEndpoint = new UriBuilder(AmqpsScheme, HostName, AmqpConstants.DefaultSecurePort).Uri;
-            HttpsEndpoint = new UriBuilder(HttpsEndpointPrefix, HostName).Uri;
+            AmqpEndpoint = new UriBuilder(IotHubConnectionStringConstants.AmqpsScheme, HostName, AmqpConstants.DefaultSecurePort).Uri;
+            HttpsEndpoint = new UriBuilder(IotHubConnectionStringConstants.HttpsEndpointPrefix, HostName).Uri;
```

Comment:
I'd recommend seeing if these can be moved to the amqp and http specific files directly.

Reply:
Same for AmqpAudience as well. Try to move it to the amqp specific class.

Reply:
Could these not be moved to the transport classes?